augur/application/schema/alembic/versions/32_add_topic_model_meta_table.py

```diff
@@ -0,0 +1,93 @@
+"""Add topic model meta table and foreign keys
+
+Revision ID: 32
+Revises: 31
+Create Date: 2025-01-27
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+from sqlalchemy import text
+
+
+# revision identifiers, used by Alembic.
+revision = '32'
+down_revision = '31'
```

Comment:
It seems like this PR has two revisions numbered 32 - is that intentional? Seems incorrect to me